docs/submission/submitting-molecular-data.md

```diff
@@ -57,78 +57,161 @@ storage.url=https://score.qa.argo.cancercollaboratory.org
 
 ## How to Upload Molecular Data
 
-### 1. Format the payload
+### Understanding Song payload fields
 
-/////// NEED THE TAB DISPLAY HERE TO SHOW DIFFEERENT STARTING PAYLOADS {SEQUENCING EXPEIRMETN FOR ONE}
+Song accepts data in JSON format, which is validated against standard JSON Schema to ensure data quality. The first step of submitting sequencing data to ARGO is to prepare SONG metadata payloads conforming to most recent JSON schema that has been defined by the DCC. The latest `sequencing_experiment` JSON Schema can be found in the ARGO [github repository](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/schemas/sequencing_experiment.json).
+
+The data fields can be broken down into four main sections: `experiment`, `sample`, `read groups` and `files`.
+
+> Note: \*\* denotes a required field. These must be provided as part of the metadata payload or it will immediately fail validation upon submission.
+
+#### **Experiment:** The experiment section contains details relevant the experimental requirements imposed during sequencing.
+
+- `**studyId`: Corresponds to your ARGO `Program ID`. This is the same Program ID that you see when you log in to the ARGO Data Platform.

```

Comment:
instead of `This is the same Program ID that you see when you log in to the ARGO Data Platform.`

Can be more specific by using the same note from the dictionary: 

This is the unique id that is assigned to your program.  If you have logged into the ARGO Data Platform, this is the Program Id that you see in the Program Services area. For example, PACA-CA is a Program ID.